apm-sniffer/apm-sdk-plugin/armeria-0.85.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/armeria/Armeria085ServerInterceptor.java

```diff
@@ -32,13 +31,15 @@
 import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.MethodInterceptResult;
 import org.apache.skywalking.apm.network.trace.component.ComponentsDefine;
 
+import java.lang.reflect.Method;
+
 @SuppressWarnings("unused") // actually used
 public class Armeria085ServerInterceptor implements InstanceMethodsAroundInterceptor {
     @Override
     public void beforeMethod(final EnhancedInstance objInst, final Method method, final Object[] allArguments,
                              final Class<?>[] argumentsTypes, final MethodInterceptResult result) {
 
-        DefaultHttpRequest httpRequest = (DefaultHttpRequest) allArguments[1];
+        HttpRequest httpRequest = (HttpRequest) allArguments[1];
```

Comment:
I think you don't need to change this. As this plugin should only work when version < 0.99. If not, it means you need another witness class in 0.85-0.99 plugin.

Reply:
CI fails due to this.